server/migrations/1584377249878-AddInterestedColumnToUserChapter.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddInterestedColumnToUserChapter1584377249878 implements MigrationInterface {
+  name = 'AddInterestedColumnToUserChapter1584377249878';
+
+  public async up(queryRunner: QueryRunner): Promise<any> {
+    await queryRunner.query(`ALTER TABLE "user_chapters" ADD "interested" BOOLEAN NOT NULL`, undefined);
```

Comment:
Is setting NOT NULL with no default value going to confuse things if there are existing records? I see the model has a default, but since this effectively a constraint at the database-level I wonder if a default is helpful and/or desirable here as well.